scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -375,10 +375,35 @@
                               (apply max))]
     utilization))
 
+(defn read-datomic-params
+  [conn]
+  (-<>>
+   (d/pull (mt/db conn) ["*"] :rebalancer/config)
+   (dissoc <> ":db/id" ":db/ident")
+   (map (fn [[k v]]
+          [(keyword (name (keyword k))) v]))
+   (into {})))
+
+(defn update-datomic-params-from-config!
+  [conn config]
+  (let [recognized-params (select-keys config [:min-utilization-threshold
```

Comment:
It is probably worth putting these as a variable at the top or pulling them from the schema so that if we add another rebalancer setting it is picked up.